### DISCUSSION TITLE
Any way to disable x64-osx-dbg during installation?

### DISCUSSION THREAD

**Discussion by agustina-laurenti** [microsoft/vcpkg#25393] [General]
About 1h ago I just tried to install llvm in macOS but it's taking too much time and disk space during "Building x64-osx-dbg" stage. Is there a way to just skip it and build directly the x64-osx-rel?

```
./vcpkg install llvm
...
-- Configuring x64-osx-dbg
-- Configuring x64-osx-rel
-- Building x64-osx-dbg   <--- takes ages
```

**Comment by Osyotr** [comment]
You need to create an overlay triplet and use per-port customization to set `VCPKG_BUILD_TYPE` for llvm.
https://github.com/microsoft/vcpkg/blob/master/docs/users/triplets.md#per-port-customization
https://github.com/microsoft/vcpkg/blob/master/docs/users/triplets.md#vcpkg_build_type

Or use community triplet `x64-osx-release`, but it will build only release versions for all your libraries.
